src/transforms/log_to_metric.rs

```diff
@@ -63,13 +63,25 @@ pub struct MetricConfig {
     pub namespace: Option<Template>,
 
     /// Tags to apply to the metric.
-    pub tags: Option<IndexMap<String, Template>>,
+    pub tags: Option<IndexMap<String, TagConfig>>,
 
     #[configurable(derived)]
     #[serde(flatten)]
     pub metric: MetricTypeConfig,
 }
 
+/// Specification of the value of a created tag. This may be a single value, a `null` for a bare
+/// tag, or an array of either.
```

Comment:
Ideally, this should be structured as:

```suggestion
/// Specification of the value of a created tag.
///
/// This may be a single string value, a `null` for a bare tag, or an array of either.
```

This will generate a more nuanced title/description in the configuration schema, as the code generation pays attention to the doc comments in the same way that `rustdoc` does, which is a lot like Git's recommended commit message format of "single line summary, line break, further detail if any".